modules/ivc_champva/app/models/ivc_champva/vha_10_7959c_rev2025.rb

```diff
@@ -65,18 +65,18 @@ def track_user_identity
     def track_current_user_loa(current_user)
       current_user_loa = current_user&.loa&.[](:current) || 0
       StatsD.increment("#{STATS_KEY}.#{current_user_loa}")
-      Rails.logger.info('IVC ChampVA Forms - 10-7959C-REV2025 rev2025 Current User LOA', current_user_loa:)
```

Comment:
`rev2025` seemed redundant, so I removed it. Please let me know if this is an issue